kafka/src/main/java/org/apache/calcite/adapter/kafka/KafkaStreamTable.java

```diff
@@ -50,7 +61,10 @@
  * implemented as a STREAM table.
  */
 public class KafkaStreamTable implements ScannableTable, StreamableTable {
+  private static final Logger LOGGER = LoggerFactory.getLogger(KafkaStreamTable.class);
+
   final KafkaTableOptions tableOptions;
+  private boolean seekFlag = false;
```

Comment:
Can we add some comments for this flag ?

Reply:
ok, I have add some note.